recipes/cmake/3.x.x/conanfile.py

```diff
@@ -99,6 +99,7 @@ def generate(self):
             save(self, "bootstrap_args", json.dumps({"bootstrap_cmake_options": ' '.join(arg for arg in bootstrap_cmake_options)}))
         else:
             tc = CMakeToolchain(self)
+            tc.variables["BUILD_TESTING"] = False
```

Comment:
```suggestion
            # Disabling testing because CMake tests build can fail in Windows in some cases
            tc.variables["BUILD_TESTING"] = False
```